src/lib/cache/subCache.cpp

```diff
@@ -734,6 +737,8 @@ void subCacheItemInsert
   cSubP->expression.geometry   = geometry;
   cSubP->expression.coords     = coords;
   cSubP->expression.georel     = georel;
+  cSubP->httpInfo.url          = url.c_str();
+  cSubP->httpInfo.verb         = POST;  // FIXME PR: Unhardwire
```

Comment:
Or maybe using `httpInfo*` as argument. If that parameter is not NULL, then the cache inserting was called from the NGSIv2 logic and it is a matter of copying into cSubP->httpInfo. If it is NULL, you can use defaults for cSubP->httpInfo.